.github/workflows/build_flang.yml

```diff
@@ -84,20 +177,43 @@ jobs:
           
       - name: Build Flang
         run: |
-          CMAKE_OPTIONS="-DLLVM_TARGETS_TO_BUILD=${{ matrix.target }} \
+          mkdir -p build && cd build
+          cmake -DLLVM_TARGETS_TO_BUILD=${{ matrix.target }} \
             -DCMAKE_INSTALL_PREFIX=/usr/local \
             -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_C_COMPILER=/usr/bin/gcc-9 \
-            -DCMAKE_CXX_COMPILER=/usr/bin/g++-9"
-          mkdir -p build && cd build
-          cmake $CMAKE_OPTIONS -DCMAKE_Fortran_COMPILER=/usr/local/bin/flang ..
+            -DCMAKE_C_COMPILER=/usr/bin/${{ matrix.cc }} \
+            -DCMAKE_CXX_COMPILER=/usr/bin/${{ matrix.cpp }} \
+            -DCMAKE_Fortran_COMPILER=/usr/local/bin/flang \
+            -DFLANG_LLVM_EXTENSIONS=ON \
+            -DCMAKE_Fortran_COMPILER_ID=Flang \
+            -DFLANG_INCLUDE_DOCS=ON \
```

Comment:
Nit: Consider sorting the last three lines so that they are in alphabetical order.

Reply:
Fixed :)